src-electron/db/zap-schema.sql

```diff
@@ -441,6 +441,7 @@ CREATE TABLE IF NOT EXISTS "DEVICE_TYPE_FEATURE" (
   "DEVICE_TYPE_CLUSTER_REF" integer,
   "FEATURE_REF" integer,
   "FEATURE_CODE" text,
+  "DEVICE_TYPE_CLUSTER_CONFORMANCE" text,
```

Comment:
Why is this over here and not in the device type cluster table?

Reply:
Maybe we should rename this, it's about DEVICE_TYPE_FEATURE_CONFORMANCE